efficientdet/hparams_config.py

```diff
@@ -249,6 +249,8 @@ def default_detection_configs():
 
   # For post-processing nms, must be a dict.
   h.nms_configs = {'method': 'hard'}
+  h.nms_score_thresh = 0.3
```

Comment:
How about put all these to nms_configs (postprocess.py is assuming these):

```
h.nms_configs = {
  'method': 'hard',
  'score_thresh': 0.5,
  'nms_max_output_size: 100,
}
```

I am also consider making this config file hierarchical like:

```
train:
  xxx

eval:
  yyy

nms:
  zzz
```

What do you think?

Reply:
Good point.